lib/http_perform.c

```diff
@@ -396,6 +395,10 @@ sf_bool STDCALL http_perform(CURL *curl,
 
     // We were successful so parse JSON from text
     if (ret) {
+        //Check if the "code" attribute exist in the response texts
+        if(!strstr(buffer.buffer, "code")){
```

Comment:
Should we search for `"code"` (with the double quotes) to eliminate false positives (ie. `code` could appear in the rowset response)